src/analyses/call_graph.cpp

```diff
@@ -35,9 +35,9 @@ call_grapht::call_grapht(
   }
 }
 
-void call_grapht::add(
-  const irep_idt &function,
-  const goto_programt &body)
+static void forall_callsites(
+  const goto_programt &body,
+  std::function<void(goto_programt::const_targett, const irep_idt &)> callback)
```

Comment:
Could you name `callback` something more descriptive e.g. `callsite_task`? And ideally add doxygen comments.